dapper/internal/health/check.go

```diff
@@ -0,0 +1,63 @@
+package health
```

Comment:
dapper has its own `go.mod` and `vendor` folder - can you not just get it from kit?